torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -39,29 +39,38 @@
 class FSDPCommContext:
     """This has the communication state shared across FSDP states/parameter groups."""
 
-    def init(self):
-        # Setting the all-gather/reduce-scatter streams to be higher priority
-        # can help avoid some issues where their copies in/out are delayed and
-        # block computation
-        high_priority = -1
+    def __init__(self):
+        # Initialize all streams using default stream at construction time and
+        # set them to new streams during lazy initialization
+        current_stream = torch.cuda.current_stream()
```

Comment:
In the future, if we want to avoid assuming that CUDA is available, we can init with a dummy stream.